src/main/java/cofh/thermaldynamics/duct/entity/Camera.java

```diff
@@ -17,7 +17,7 @@ public Camera() {
 		super(null);
 		this.width = 0;
 		this.height = 0;
-		invulnerable = true;
+		this.setEntityInvulnerable(true);
```

Comment:
There's not really any inherit benefit to avoiding AT usage. In fact, it's slower since there's a method overhead to it during gameplay.

My guess is that the setter flat out didn't exist during 1.7 though.